micropub.php

```diff
@@ -279,7 +280,15 @@ private static function generate_args() {
     if (isset($_POST['url'])) {
       $args['ID'] = url_to_postid($_POST['url']);
     }
-
+    // perform these functions only for creates
+    if (!isset($args['ID'])) {
+      if (isset($args['post_title'])&&!isset($args['post_name'])) {
```

Comment:
again, style, spaces around operators like `&&`, `=` (below), etc. you'll want to watch out for this in the future. :P